crypto/bls_multisig.go

```diff
@@ -482,50 +485,60 @@ func BatchVerifyBLSSignaturesOneMessage(
 	// set BLS context
 	blsInstance.reInit()
 
+	// boolean array returned when errors occur
+	falseSlice := make([]bool, len(sigs))
+
 	// empty list check
 	if len(pks) == 0 {
-		return []bool{}, fmt.Errorf("invalid list of public keys: %w", blsAggregateEmptyListError)
+		return falseSlice, fmt.Errorf("invalid list of public keys: %w", blsAggregateEmptyListError)
 	}
 
 	if len(pks) != len(sigs) {
-		return []bool{}, invalidInputsErrorf(
+		return falseSlice, invalidInputsErrorf(
 			"keys length %d and signatures length %d are mismatching",
 			len(pks),
 			len(sigs))
 	}
 
-	verifBool := make([]bool, len(sigs))
+	returnBool := make([]bool, len(sigs))
+	for i := range returnBool {
+		returnBool[i] = true // default to true
+	}
```

Comment:
Initialized too early, it's not needed until L519. 

Could be moved down to L519 where it's it's going to be used

Reply:
https://github.com/onflow/flow-go/pull/4225/commits/e96aa5963a98c3407dc2ce2833ff40e965a42aaf